### DISCUSSION TITLE
New features ideas (iCal, filter rules, anti tracker, UI improvements)

### DISCUSSION THREAD

**Discussion by miditkl** [tutao/tutanota#6909] [Ideas]
Heyy, well I subscribed an annual revolutionary plan and I discovered some features I missed:

Calendar:
1. iCal URL import with auto syncing (and also a question, why is there a "Other calendars" field?)

Mail:
1. Better Mail filtering (like Protonmail). I like the way how you can add multiple mails / multiple If statements.
2. An option to immediatly create a filter for the mail I've got (which actually just autofills a filter form) and move it into the folder when I want to
3. Also after I created a filter, that it will sort it for me and apply my filter rules to all my mails (if I say so)
4. An anti-tracker feature + An overview if trackers were in use

And maybe a better UI for the settings, as it looks more technical and like "experimental".... It needs to be adjusted to the "usual" design you use for the mail etc. 

I like tuta.com so far but these are features that could be very important especially for me 😅

**Comment by charlag** [comment]
Hey, thank you for your feedback!

Calendar:

1. I think it's coming eventually. Other calendars are for shared calendars

Mail:

1. We will check that. You can simulate most of the things with regular expressions but it's not user-friendly
2. You can create a filter if you click on the sender bubble.
3. If you mark emails as unread and reopen the folder the filters will be re-applied.

**Comment by Backspaze** [comment]
Regarding iCAL URL import, there's an issue open for that now https://github.com/tutao/tutanota/issues/7210.

**Comment by miditkl** [comment]
> If you mark emails as unread and reopen the folder the filters will be re-applied.

@charlag This workaround is not working anymore...

**Reply by hrb-hub** [reply]
Things have changed recently, but you can still re-apply inbox rules (filters) by selecting the mails you wish to sort then clicking **`Reapply inbox rules`** from the more actions menu (please note that this is only possible in the inbox folder).

<img width="300"  alt="" src="https://github.com/user-attachments/assets/440cad79-df00-46c4-bac7-73134188b79e" />

Hope this helps :)
